infection.json5.dist

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "vendor/infection/infection/resources/schema.json",
+    "testFrameworkOptions": "--testsuite=unit",
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "mutators": {
+        "@default": true,
+        "Throw_": {
+            "ignore": [
+                "PhpCsFixer\\Cache\\FileHandler"
```

Comment:
as it's json5 (nice! not many tools allow for that format!) we can add comment describing why we are ignoring this class

Reply:
Added explanation. I should ignore some mutants for `src/Runner/Runner.php` too, but did not find out which one yet. In terms of `Runner`, mutation tests "only" causes modifications of fixture files, but when it comes to `FileHandler` it was modifying file permissions in a way it required `sudo` to get it back (and unreadable file was causing failures in tests).